refactor: update docs of K8s deployment (#1140)
GreptimeTeam/docs: docs/user-guide/operations/deploy-on-kubernetes/greptimedb-operator.md

```shell
helm upgrade \
  --install greptimedb-operator greptime/greptimedb-operator \
  --create-namespace \
  -n greptimedb-admin
```

## Install and upgrade CRDs

Helm cannot upgrade custom resource definitions in the `<chart>/crds` folder [by design](https://helm.sh/docs/chart_best_practices/custom_resource_definitions/#some-caveats-and-explanations).

> We support installing and upgrading CRDs automatically using the chart. You can disable this behavior by using `--set crds.install=false` when installing the chart:

Reviewer: ```suggestion
GreptimeDB support installing and upgrading CRDs automatically using the chart.
```

```shell
helm upgrade \
  --install greptimedb-operator greptime/greptimedb-operator \
  --create-namespace \
  -n greptimedb-admin
```

> When you uninstall the release, it will not delete the CRDs by default.

Reviewer: ```suggestion
```

```shell
helm upgrade \
  --install greptimedb-operator greptime/greptimedb-operator \
  --create-namespace \
  -n greptimedb-admin
```

## Installation and upgrade of CRDs

Helm cannot upgrade custom resource definitions in the `<chart>/crds` folder [by design](https://helm.sh/docs/chart_best_practices/custom_resource_definitions/#some-caveats-and-explanations).

> Starting from the **v0.2.1 version** of the GreptimeDB Operator chart, we support installing and upgrading CRDs automatically using the chart. You can disable this behavior by using `--set crds.install=false` when installing the chart:

Author: I already remove the unnecessary NOTE.

```shell
helm upgrade \
  --install greptimedb-operator greptime/greptimedb-operator \
  --create-namespace \
  -n greptimedb-admin

> ```

Author: No, we don't need to set it. If you want to disable crd installation and upgrade, you must set the config. We hope our users install CRDs by default.
